src/server/controllers/users.controller.js

```diff
@@ -22,7 +22,7 @@ export function getAll(req, res, next) {
 // Get all user list except logged in user
 export function getMentorList(req, res, next) {
   // TODO: Once session_auth is implemented, replace the email address with req.session.email.
-  User.find({ 'email' : { $ne : 'session@yoda.com' }},
+  User.find({ email: { $ne: 'session@yoda.com' } },
```

Comment:
You'll have a conflict with @SarahhhC 's PR. Please ensure to update before submit (whoever submits after :)